examples/vision/barlow_twins.py

```diff
@@ -0,0 +1,1101 @@
+"""
+Title: Barlow Twins
+Author: [Abhiraam Eranti](https://github.com/dewball345)
+Date created: 11/4/21
+Last modified: 11/17/21
+Description: A keras implemenation of Barlow Twins(constrastive SSL with redundancy reduction).
+"""
+"""
+Paper:
+[Barlow Twins: Self-Supervised Learning via Redundancy
+Reduction](https://arxiv.org/abs/2103.03230)
```

Comment:
Keep the markdown hyperlinks on the same line. Put these 2 lines into the same line.